src/styles/index.scss

```diff
@@ -1,3 +1,116 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: solid 1px #000;
+  border-radius: 5px;
+}
+
+.header {
+  margin: 32px 19px 40px;
```

Comment:
use paddings for card instead of margins for other elements.

Reply:
done